prometheus/exporter/watchdog.py

```diff
@@ -125,12 +125,18 @@ def ssh_exec(host_config, command, histogram=ssh_histogram):
         port = 22
         if "sshport" in host_config:
             port = int(host_config["sshport"])
+        key_filename = None
+        if 'keyfile_path' in host_config:
+            if os.path.isfile(str(host_config['keyfile_path'])) and host_config['keyfile_path'] is not None:
+                key_filename = str(host_config['keyfile_path'])
+            else:
+                logger.warn("The key file: {0} specified doesn't exist".format(host_config['keyfile_path']))
```

Comment:
please use `logger.error` and increase `error_counter` like `error_counter.labels(type="ssh_key_file").inc()`